src/libtest/lib.rs

```diff
@@ -653,13 +658,13 @@ pub fn parse_opts(args: &[String]) -> Option<OptRes> {
     Some(Ok(test_opts))
 }
 
-#[derive(Clone, PartialEq)]
+#[derive(Clone, PartialEq, RustcEncodable, RustcDecodable)]
```

Comment:
I think it's probably safe to assume that benchmarks don't need to run in separate processes because they typically never fail, but was this new annotation needed for a different reason?